File: ios/filtering-content/GetItemsOfTypes.swift
// DocSection: filtering_get_items_of_types
// Gets items of following content types
let customQuery = "items?system.type[in]=product,article,news"

client.getItems(modelType: Article.self, customQuery: customQuery) { (isSuccess, itemsResponse, error) in
    if isSuccess {
        if let articles = itemsResponse?.items {
            // Use your items here
        }
    } else {
        if let error = error {
            print(error)
        }
    }
}
// EndDocSection

File: java/filter-content/GetItemsOfTypes.java
// DocSection: filtering_get_items_of_types
// Gets items based on the types Article, Product, and News
CompletionStage<ContentItemsListingResponse> items = client.getItems(
  DeliveryParameterBuilder.params()
    .filterIn("system.type", "article", "product", "news")
    .build()
);
// EndDocSection
